src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -131,25 +139,27 @@ private CallbackResponse handleAdmitPartOfClaim(CallbackParams callbackParams) {
 
     private CaseData populateRespondentResponseTypeSpecPaidStatus(CaseData caseData) {
         if (SpecJourneyConstantLRSpec.HAS_PAID_THE_AMOUNT_CLAIMED.equals(caseData.getDefenceRouteRequired())
-            && caseData.getRespondToClaim().getHowMuchWasPaid() != null
-            && caseData.getRespondToClaim().getHowMuchWasPaid().compareTo(caseData.getTotalClaimAmount()) < 0) {
-            caseData = caseData.toBuilder()
-                .respondent1ClaimResponsePaymentAdmissionForSpec(
-                    RespondentResponseTypeSpecPaidStatus.PAID_LESS_THAN_CLAIMED_AMOUNT).build();
-        } else if (SpecJourneyConstantLRSpec.HAS_PAID_THE_AMOUNT_CLAIMED
-            .equals(caseData.getDefenceRouteRequired())
-            && caseData.getRespondToClaim().getHowMuchWasPaid() != null
-            && caseData.getRespondToClaim().getHowMuchWasPaid().compareTo(caseData.getTotalClaimAmount()) >= 0) {
-            caseData = caseData.toBuilder()
-                .respondent1ClaimResponsePaymentAdmissionForSpec(
-                    RespondentResponseTypeSpecPaidStatus.PAID_FULL_OR_MORE_THAN_CLAIMED_AMOUNT).build();
+            && caseData.getRespondToClaim().getHowMuchWasPaid() != null) {
+            int comparison = caseData.getRespondToClaim().getHowMuchWasPaid()
+                .compareTo(caseData.getTotalClaimAmount().multiply(BigDecimal.valueOf(100)));
```

Comment:
Instead of multiplying by 100 could you please use MonetaryConversions class to convert pounds to pennies.